src/components/SignIn.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { Formik, Form, Field, ErrorMessage } from "formik";
+import { Link, Alert, Snackbar } from "@mui/material";
+import PropTypes from "prop-types";
+
+import INIT from "config/constants/initial";
+import submit from "api/submitSignIn";
+import validateSignIn from "utils/validateSignIn";
+import styles from "pages/styled";
+
+const SignInForm = ({ submitAutorization, handleAlertClose, alertOpen, setAlertOpen }) => (
+  <div>
+    <h2 style={styles.aboutTitle}>Authorization</h2>
+    <Formik
+      initialValues={INIT.AUTH_VALUES}
+      onSubmit={(values) => submit(values, submitAutorization, setAlertOpen)}
+      validate={validateSignIn}
+    >
+      <Form style={styles.form}>
+        <div style={styles.formBlock}>
+          <label style={styles.formLabel} htmlFor="email">
+            Email
+          </label>
+          <Field style={styles.formInput} type="email" id="email" name="email" />
+          <ErrorMessage name="email">{(ErrorText) => <div className="error">{ErrorText}</div>}</ErrorMessage>
+        </div>
+        <div style={styles.formBlock}>
+          <label style={styles.formLabel} htmlFor="password">
+            Password
+          </label>
+          <Field style={styles.formInput} type="password" id="password" name="password" />
+          <ErrorMessage name="password">{(ErrorText) => <div className="error">{ErrorText}</div>}</ErrorMessage>
```

Comment:
Do we really need this arrow function?
Why we can't use just:
`<ErrorMessage name="password"><div className="error">{ErrorText}</div></ErrorMessage>`

Reply:
I've tried, but it doesn't work without arrow function :( 
According to formik documentation attribute "validate" of "Formik" component returns object with errors and passes each of them to "Error message" component with corresponding name. And then we have to use arrow function to work with that message